static/app/components/replays/actionCategory.tsx

```diff
@@ -34,6 +34,11 @@ function getActionCategoryInfo(crumb: Crumb): ActionCategoryInfo {
         title: t('Error'),
         description: `${crumb.category}: ${crumb.message}`,
       };
+    case BreadcrumbType.SESSION:
+      return {
+        title: t('Replay Start'),
+        description: crumb.data?.url,
+      };
```

Comment:
slightly re-purposed `SESSION` and making the assumption that it's our injected start crumb.

Reply:
Would it be better to create a new type?

Reply:
`BreadcrumbType.INIT`